charts/kuberpult/values.yaml.tpl

```
@@ -264,3 +264,9 @@ revolution:
     token: ""
     # Maximum number of requests send in parallel.
     concurrency: 20
+
+# Whether the rollout service should self-manage applications
+manageArgoApplications:
+  enabled: false
+  # List of teams that should be self managed by the rollout service. The apps that belong to the teams in the filter.
```

either remove this sentence or rephrase it.

```suggestion
  # List of teams that should be self managed by the rollout service.
```